api/app/signals/apps/email_integrations/actions/__init__.py

```diff
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: MPL-2.0
 # Copyright (C) 2021 - 2022 Gemeente Amsterdam
```

Comment:
Nitpick and not blocking but the SPDX header should now also contain `, Delta10 B.V.`

Reply:
I added the missing SPDX headers to the files.